specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/MultivariateAnomalyDetector.json

```
@@ -0,0 +1,1119 @@
+            "description": "Result identifier",
+            "format": "uuid"
+          }
+        ],
+        "x-ms-examples": {
+          "Get detection result": {
+            "$ref": "./examples/GetResult.json"
+          }
+        }
+      }
+    },
+    "/multivariate/models": {
+      "post": {
+        "responses": {
+          "201": {
+            "description": "Create and train a multivariate anomaly detection model.",
+            "headers": {
+              "Location": {
+                "description": "Location and ID of the model.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "Error code",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Train a Multivariate Anomaly Detection Model",
+        "description": "Create and train a multivariate anomaly detection model. The request must include a source parameter to indicate an externally accessible Azure storage Uri (preferably a Shared Access Signature Uri). All time-series used in generate the model must be zipped into one single file. Each time-series will be in a single CSV file in which the first column is timestamp and the second column is value.",
+        "operationId": "TrainMultivariateModel",
+        "parameters": [
+          {
+            "name": "body",
+            "required": true,
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/ModelInfo"
+            },
+            "description": "Training request"
+          }
+        ],
+        "x-ms-examples": {
+          "Train Multivariate model": {
+            "$ref": "./examples/TrainModel.json"
+          }
+        }
+      },
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Succeed to get the list of models.",
+            "schema": {
+              "$ref": "#/definitions/ModelList"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "Error code",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "List Multivariate Models",
+        "description": "List models of a subscription",
+        "operationId": "ListMultivariateModel",
+        "parameters": [
+          {
+            "type": "integer",
+            "default": 0,
+            "required": false,
+            "description": "$skip indicates how many models will be skipped.",
+            "name": "$skip",
```

The [Azure API Guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#query-options) are to drop the "$" from the "top" and "skip" parameter names.
```suggestion
            "description": "skip indicates how many models will be skipped.",
            "name": "skip",
```

---

Should we follow the previous API version as this may cause breaking changes for our customer.